metricbeat/module/jolokia/jmx/_meta/data.json

```diff
@@ -1,33 +1,34 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
+    "event": {
+        "dataset": "jolokia.testnamespace",
+        "duration": 115000,
+        "module": "jolokia"
```

Comment:
I think this is not expected to change in 6.X, could you regenerate the data.json with this version?

Reply:
Yep :-) Will do!

Reply:
Or leave it as it was in master.